apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -52,32 +52,81 @@ When considering compute upgrades, assess whether your bottlenecks are hardware-
+
+For example, the micro plan can burst up to 2,085 Mbps for 30 minutes a day and reverts to the baseline performance of 87 Mbps. Your disk IO budget gets replenished throughout the day.
+
+### Dedicated I/O Resources (Larger Instances)
+
+Larger instances, such as 4XL and above, come with dedicated I/O resources and don't rely on I/O balance or burst capacity. These instances are designed for sustained, high performance with specific IOPS and throughput limits. They provide consistent performance without needing to burst. If you hit your IOPS or throughput limit, throttling will occur, but you won't have burst credits to exceed these limits.
+
+### Storage Volume Types: gp3 vs. io2
+
+Here’s how **gp3** and **io2** storage volumes differ:
+
+**gp3 (General Purpose SSD) (default)** This is a versatile and cost-effective option, perfect for a range of everyday workloads that need moderate IOPS and throughput. It supports volume sizes from 1GB to 16TB. You get IOPS ranging from 3,000 (default) up to 16,000 (maximum), and throughput from 125 MB/s (default) to 1,000 MB/s (maximum). It’s well-suited for general tasks like boot volumes and small to medium-sized databases.
+
+**io2 (Provisioned IOPS SSD)** Designed for high-demand scenarios where very high IOPS and low latency are crucial. It supports volumes ranging from 4GB to 16TB. With IOPS reaching up to 256,000 (depending on your instance type) and throughput scaling with IOPS, io2 is designed for large-scale databases and mission-critical applications. While it offers top-tier performance, it does come with a higher price tag.
+
+If you’re choosing between these two, think about your workload needs to find the best fit.
+
+### Choosing the Right Compute Add-On for Consistent Disk Performance
+
+If you need consistent disk performance, choose the 4XL or larger compute add-on which has the same baseline and maximum disk throughput and IOPS.
 
 If you're unsure of how much throughput or IOPS your application requires, you can load test your project and inspect these [metrics in the Dashboard](https://supabase.com/dashboard/project/_/reports). If the `Disk IO % consumed` stat is more than 1%, it indicates that your workload has burst beyond the baseline IO throughput during the day. If this metric goes to 100%, the workload has used up all available disk IO budget and will revert to baseline performance. Projects that use any disk IO budget are good candidates for upgrading to a larger compute add-on with higher baseline throughput.
 
+<Admonition type="information">
+
+Compute add-on changes will not cause you to lose your settings.
```

Comment:
`lose your disk attributes` (use `attributes`/`settings`/`configuration` consistently to avoid confusion)